components/caseStudyDetails/CaseStudyDetailRoleTechStack.tsx

```diff
@@ -34,7 +34,7 @@ const CaseStudyDetailRoleTechStack = ({ project }: { project: string }) => {
             </div>
           </div>
           <div className="flex flex-row flex-wrap gap-x-5 gap-y-[2.06rem] xl:grid xl:grid-cols-7 xl:gap-x-[2.32rem] xl:gap-y-[2.1rem]">
-            {projectDetail.techStack.map((Tech) => (
+            {projectDetail?.techStack.map((Tech) => (
```

Comment:
Why is Tech capitalized? In JS/TS we usually follow the camelCase convention where names start with lowercase. An exception to this is usually when you create a functional component. There it's seen as fine to start with a capital letter.